src/seedu/addressbook/AddressBook.java

```diff
@@ -483,10 +485,12 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      * @return list of persons in full model with name containing some of the keywords
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
+        final Collection<String> normalizedKeywords = keywords.stream().map(keyword -> keyword.toLowerCase()).collect(Collectors.toSet());
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> normalizedWordsInName = wordsInName.stream().map(keyword -> keyword.toLowerCase()).collect(Collectors.toSet());
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.